.pre-commit-hooks.yaml

```diff
@@ -1,10 +1,20 @@
--   id: ruff
-    name: ruff
-    description: "Run 'ruff' for extremely fast Python linting"
-    entry: ruff check --force-exclude
-    language: python
-    'types_or': [python, pyi]
-    args: []
-    require_serial: true
-    additional_dependencies: []
-    minimum_pre_commit_version: '2.9.2'
+- id: ruff
+  name: ruff
+  description: "Run 'ruff' for extremely fast Python linting"
+  entry: ruff check --force-exclude
+  language: python
+  "types_or": [python, pyi]
+  args: []
+  require_serial: true
+  additional_dependencies: []
+  minimum_pre_commit_version: "2.9.2"
+- id: ruff-format
+  name: ruff-format
+  description: "Run 'ruff format' for extremely fast Python formatting"
+  entry: ruff format --force-exclude
+  language: python
+  "types_or": [python, pyi]
+  args: []
+  require_serial: true
+  additional_dependencies: []
+  minimum_pre_commit_version: "2.9.2"
```

Comment:
What was this formatted with?

Reply:
Good question I didn't even notice it formatted — I think the RedHat YAML Extension in VSCode. Happy to revert or use something else.

Reply:
The other formatting was weird actually just going to merge with this — we can adjust later if desired.